packages/fxa-auth-server/lib/metricsCache.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { RedisShared } = require('fxa-shared/db/redis');
+const { Container } = require('typedi');
+const { AuthLogger } = require('./types');
+const { StatsD } = require('hot-shots');
+
+function resolveLogger() {
+  if (Container.has(AuthLogger)) return Container.get(AuthLogger);
+}
+function resolveMetrics() {
+  if (Container.has(StatsD)) {
+    return Container.get(StatsD);
+  }
+}
+
+export class MetricsRedis extends RedisShared {
+  constructor(config) {
+    super(config.redis.metrics, resolveLogger(), resolveMetrics());
+    this.enabled = config.redis?.metrics?.enabled || true;
```

Comment:
This will always evaluate to true. Did you want to explicitly check for null/undefined?